LiveSplit.TR3.asl

```diff
@@ -144,19 +144,32 @@ reset
 
 split
 {
-    // Determine if a split should occur.
     bool shouldSplit;
-    if (current.level == 19)
+    if (current.level == 19 && old.level == 19)
+    {
+        // While playing the "last" level (regardless of IL/FG and if the bonus level, All Hallows, will be played)
```

Comment:
Imho this is a bit too verbose: "Last level: split when player's control ends and helicopter scene begins." should suffice.